skelebot/systems/generators/dockerfile.py

```diff
@@ -5,11 +5,11 @@
 
 FILE_PATH = "{path}/Dockerfile"
 
-PY_INSTALL = "RUN [\"pip\", \"install\", \"{dep}\"]\n"
-PY_INSTALL_VERSION = "RUN [\"pip\", \"install\", \"{depName}=={version}\"]\n"
-PY_INSTALL_GITHUB = "RUN [\"pip\", \"install\", \"git+{depPath}\"]\n"
+PY_INSTALL = "RUN [\"pip3\", \"install\", \"{dep}\"]\n"
+PY_INSTALL_VERSION = "RUN [\"pip3\", \"install\", \"{depName}=={version}\"]\n"
+PY_INSTALL_GITHUB = "RUN [\"pip3\", \"install\", \"git+{depPath}\"]\n"
 PY_INSTALL_FILE = "COPY {depPath} {depPath}\n"
-PY_INSTALL_FILE += "RUN [\"pip\", \"install\", \"/app/{depPath}\"]\n"
+PY_INSTALL_FILE += "RUN [\"pip3\", \"install\", \"/app/{depPath}\"]\n"
```

Comment:
Sorry this just went through, I forgot to "submit" the review.

Hopefully my last comment makes more sense now. ;)

Reply:
No worries. You can feel free to open this is a PR for review (not just a draft) and keep working on it. That should give you the hacktoberfest credit. You deserve it, you picked up the most complex issue we had on the project.

Reply:
thanks! Also I was going to ask a question about python3 bump from python. Is it possible to modify the python base image to default to python3? They're sunsetting python2 officialy in January. https://www.python.org/doc/sunset-python-2/

Reply:
Also please tell if the new test is good or if you'd like more tests elsewhere.

Reply:
The base image for Python that we are using is actually using Python3, it simply doesn't have Python2 installed, so the `python` command on the base image will execute Python 3.

Unfortunately the R base image is different, and requires you to specifically use the `python3` command to use Python3 instead of Python2.

Reply:
Note: the same is true for `pip` and `pip3`.

Reply:
Added a second variant of these variables for R+Python